requirements.txt

```diff
@@ -1,4 +1,6 @@
 langchain-core==0.2.40
 langchain-community==0.2.17
 google-cloud-firestore==2.18.0
+google-cloud-storage==2.18.2
```

Comment:
These deps need to also be defined in the [pyproject.toml](https://github.com/googleapis/langchain-google-firestore-python/blob/265957288df57f33e1ace3074b4d0d8097e930fe/pyproject.toml#L11)

Reply:
I see dependency ranges in that file:

```
dependencies = [
    "langchain-core>=0.1.1, <1.0.0",
    "langchain-community>=0.0.18, <1.0.0",
    "google-cloud-firestore>=2.16.0, <3.0.0",
    "more_itertools>=10.2.0, <11.0.0"
]
```

But I'm not sure what range I should use for `google-cloud-storage`, any idea?

Reply:
I would use the current version we've tested against unless we want to try to test against a lower version. Then cap it at the next major release like "google-cloud-storage>=2.18.2, <3.0.0"

Reply:
Ok, added this now